rpc/cloudagent/admin/logging/moc_cloudagent_logging.proto

```diff
@@ -26,3 +28,19 @@ message LogFileResponse {
 	string                      error    = 3;
 	string                      filename = 4;
 }
```

Comment:
curious - why are we making changes in both. Isnt both the agents using the common proto

Reply:
Yes madhan , actually the set service , having setrequest for cloudagent and nodeagent have different parameters

Reply:
So Madhan, Already there were changes for logging in both cloudagent proto and common proto, moc-sdk  was using cloudagant proto and wssd-sdk was using common proto for logging. So that's why I have been consistent with the way which was already there.

Reply:
okay, lets refactor this later.